### ISSUE TITLE
Error in hacking mission minigame

### ISSUE THREAD

**Issue by Umezawa94** [danielyxie/bitburner#185]
```
ERROR Parsing Hacking Mission Node Id. Could not find coordinates
bundle.js:37147:9
TypeError: targetNode is null
bundle.js:37472:1
```
Seems to happen when dragging the connection to an invalid node. The error seems to interrupt the tick cycle, so after it occurs, none of the nodes in the minigame, nor the values outside of it, like money, refresh anymore.

**Comment by danielyxie** [comment]
What exactly do you mean by an invalid node? I can't seem to reproduce this issue

**Comment by Umezawa94** [comment]
Seems like it only happens when dragging an existing connection over to an invalid target. Also the node needs to be attacking or scanning for the issue to occur.
![grafik](https://user-images.githubusercontent.com/9123495/36413543-2cf7dc12-161f-11e8-9909-45bea4a9b925.png)
This drag would trigger it for me. Looking back, I wonder how I managed to trigger it so frequently.

**Comment by sordidfellow** [comment]
It happens to me about 50-80% of the time in the minigame as well.  Have to reload the browser page to get the game to continue to function.

Since you can't drag more than one core at a time, though, the minigame is pretty rough.  I expected selecting all nodes and dragging from any one of them to point them all at a target , but no...

**Comment by danielyxie** [comment]
Should be fixed in v0.34.5. If not you can reopen and let me know.

I'll put your suggestion on my todo sordid
